src/components/App/style.scss

```diff
@@ -86,9 +73,6 @@
 .settings-wrap {
     position: relative;
     display: inline-block;
-    float: right;
-    margin-right: 56px;
-    top: 16px;
-    color: #FFFFFF;
+    color: #111111;
```

Comment:
is there a const value for that color that you can use from other SCSS imports?

Reply:
I'll keep this note till I get the final design colors.